unit_tests/efifeatures.h

```diff
@@ -76,3 +76,5 @@
 #define EFI_FUEL_PUMP TRUE
 
 #define EFI_LUA TRUE
+
+#define EFI_VEHICLE_SPEED TRUE
```

Comment:
we already have it in https://github.com/rusefi/rusefi/blob/master/unit_tests/efifeatures.h#L16